plugins/widget/plugin.js

```diff
@@ -3436,9 +3436,10 @@
 					scrollTop;
 
 				copyBin.setHtml(
+					CKEDITOR.tools.htmlSafeByReview(
 					'<span data-cke-copybin-start="1">\u200b</span>' +
 					html +
-					'<span data-cke-copybin-end="1">\u200b</span>' );
+					'<span data-cke-copybin-end="1">\u200b</span>', 'This function is only used once internally with getting clipboard html. The html gotten from the clipboard is filtered using editor.dataProcessor.toDataFormat( selectedHtml )') );
```

Comment:
Is `copyBin` ever attached to the main document?

Maybe safer to do a legacy conversion if we can't guarantee that the `editor.dataProcessor.toDataFormat` will stay somewhere in the chain before this callsite.

Reply:
I agree, I'll legacy convert this with a comment above on our reasoning.

Reply:
Legacy converted